vignettes/getting_started.Rmd

```diff
@@ -150,7 +150,19 @@ str(inputDat)
 
 To do this, we first need to get the information from our IQB data bank via ZKDeamon, or alternatively via SPSS. 
 
-#### readDeamonXlsx
+#### ZKDeamon
+
+ZKDeamon is a program used by IQB that can be found in the IQB internal folders (i:). After installing you can get the meta data and the items from your specific study using information stored in the IQB Databases (DB2/DB3/DB4). Alternatively you can get the meta data from an SPSS file (see below). You can produce Excel files, which are expected to have the following sheets: “units”, “subunits”, “values”, “unitrecoding”, “sav-files”, “params”, “aggregate-missings”, “itemproperties”, “propertylabels”, “booklets”, and “blocks”. 
```

Comment:
I didn't add how to install ZKDeamon or how to use it. I can imagine a link to the program would be nice, but as it's internal, I wasn't sure what to write. any suggestions?